build/CMakeLists.txt

```diff
@@ -201,6 +202,49 @@ ExternalProject_Add(prometheus-cpp
     -DCMAKE_INSTALL_PREFIX:PATH=${CMAKE_CURRENT_BINARY_DIR}/prometheus-cpp/install
 )
 
+#
+# Build crc32c
+#
+ExternalProject_Add(crc32c
+  PREFIX crc32c
+  GIT_REPOSITORY "https://github.com/google/crc32c.git"
+  GIT_TAG "1.0.6"
+  SOURCE_DIR "${CMAKE_CURRENT_BINARY_DIR}/crc32c/src/crc32c"
+  CMAKE_CACHE_ARGS
+    -DBUILD_SHARED_LIBS:STRING=yes
+    -DCRC32C_BUILD_TESTS:BOOL=OFF
+    -DCRC32C_BUILD_BENCHMARKS:BOOL=OFF
+    -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON
+    -DCMAKE_INSTALL_PREFIX:PATH=${CMAKE_CURRENT_BINARY_DIR}/crc32c/install
+)
+
+set(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} 
```

Comment:
Why is this needed? You should be able to use just -DProtobuf_DIR, -DgRPC_DIR and -Dc-ares_DIR in CMAKE_CACHE_ARGS. Doing it this way changes the path for all the projects, not just GCS.